packages/sanity/src/core/preview/documentPreviewStore.ts

```diff
@@ -114,7 +114,7 @@ export interface DocumentPreviewStoreOptions {
  * This comes with an extra memory and initial transfer cost, but gives faster updating previews and less likelihood of displaying
  * out-of-date previews as documents will be kept in sync by applying mendoza patches, instead of re-fetching preview queries
  * */
-const PREVIEW_FETCH_FULL_DOCUMENTS = false
+const PREVIEW_FETCH_FULL_DOCUMENTS = true
```

Comment:
According to your comment 
>and the commit that enables full-document-previews should likely be removed before merging into main

wdyt it we add here a TODO

```suggestion
// TODO: Remove before merging to main, see https://github.com/sanity-io/sanity/pull/7257/
const PREVIEW_FETCH_FULL_DOCUMENTS = true
```

Reply:
Good point!